### PR TITLE
Doc: Add pipeline improvements to release notes

### PR DESCRIPTION
**Pull request by karenzone**
Preview won't be available until we add 8.3 doc builds to conf file.

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -32,6 +32,10 @@ coming[8.3.0]
 [[notable-8.3.0]]
 ==== Performance improvements and notable issues fixed
 
+* {ls} is more efficient at fetching pipelines. 
+When a {ls} instance sends its pipeline IDs to {es} or central pipeline management in {kib}, it gets back only the pipeline configs that belong to that instance. 
++ 
+These improvements dramatically decrease network load while also giving users the ability to control pipelines dynamically using wildcards.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:38]: ToDo: Add links to appropriate PR number(s).

**Comment by karenzone** [comment]
@mashhurs, here's a first pass at the release notes item we discussed. Please LMKWYT.

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (mashhurs) [reply docs/static/releasenotes.asciidoc:38]: Here are the PRs:
- [ES PR](https://github.com/elastic/elasticsearch/pull/85847)
- [LS PR](https://github.com/elastic/logstash/pull/14076)

**Comment by mashhurs** [comment]
> @mashhurs, here's a first pass at the release notes item we discussed. Please LMKWYT.

That note is really clean! LGTM!

**Review by mashhurs — APPROVED** [review]
👍

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -32,6 +32,10 @@ coming[8.3.0]
 [[notable-8.3.0]]
 ==== Performance improvements and notable issues fixed
 
+* {ls} is more efficient at fetching pipelines. 
+When a {ls} instance sends its pipeline IDs to {es} or central pipeline management in {kib}, it gets back only the pipeline configs that belong to that instance.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:36]:
```suggestion
* At 8.3,  we made {ls} more efficient at fetching pipelines. 
When a {ls} instance sends its pipeline IDs to {es} or central pipeline management in {kib}, it gets back only the pipeline configs that belong to that instance. 
These enhancements required changes to both {ls} https://github.com/elastic/logstash/pull/14076[#14076] and {es} https://github.com/elastic/elasticsearch/pull/85847[#85847].
```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -32,6 +32,11 @@ coming[8.3.0]
 [[notable-8.3.0]]
 ==== Performance improvements and notable issues fixed
 
+* {ls} is more efficient at fetching pipelines as of 8.3. 
+When a {ls} instance sends its pipeline IDs to {es} or central pipeline management in {kib}, it gets back only the pipeline configs that belong to that instance. 
+These enhancements required changes to both {ls} https://github.com/elastic/logstash/pull/14076[#14076] and {es} https://github.com/elastic/elasticsearch/pull/85847[#85847].
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:37]:
```suggestion
These enhancements required changes to both {ls} https://github.com/elastic/logstash/pull/14076[(#14076)] and {es} https://github.com/elastic/elasticsearch/pull/85847[(#85847)].
```

**Comment by karenzone** [comment]
Note: The "usual" workflow is for released PRs to be picked up when the release manager generates our draft release notes.  However, we wanted to be sure that this enhancement was documented properly, so we added it early.

**Comment by karenzone** [comment]
@logstashmachine backport main

**Comment by karenzone** [comment]
@logstashmachine backport main

**Pull request merged**